build.sbt

```diff
@@ -145,22 +146,25 @@ lazy val microsite = project.module
         |<p>&copy; 2018 <a href="https://github.com/scalaz/scalaz-zio">ZIO Maintainers</a></p>
         |""".stripMargin
     ),
-    micrositeName := "ZIO",
-    micrositeDescription := "ZIO",
+  /*  micrositeCssDirectory := "/microsite/resources/microsite/css/main.css",*/
+
+    micrositeName := "",
+    micrositeDescription := "Type-safe, compositional concurrency for Scala",
     micrositeAuthor := "ZIO contributors",
-    micrositeOrganizationHomepage := "https://github.com/scalaz/scalaz-zio",
+    micrositeOrganizationHomepage := "http://localhost:4000",
```

Comment:
Looks, like original url should be there instead localhost

Reply:
tnx)